.envs/README.md

```
@@ -0,0 +1,71 @@
+- `.env.development.template` - Development template with placeholder values
+- `.env.development.local` - Active development configuration (generated with secure passwords)
+- `.env.production` - Production environment template
+
+## Usage
+
+### Development Setup
+
+1. **Initial Setup**: The development environment is already configured with secure passwords in `.env.development.local`
+
+2. **New Development Setup**: If you need to create a new development environment:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)